### DISCUSSION TITLE
Any way to disable tap/click to set volume on volume slider?

### DISCUSSION THREAD

**Discussion by dolanjp** [music-assistant/support#3186] [Feature requests and ideas]
The Music Assistant UI is great on a mobile device with one caveat - it's very easy to accidentally tap some position along the volume slider and have the volume jump extremely loud unexpectedly and possibly damage speakers. This is especially true since the incremental volume increase button is right next to the top end of the volume slider.

Is there any way to disable tapping/clicking on the volume slider and only allow dragging from the current volume position?

![IMG_4827](https://github.com/user-attachments/assets/2439895a-49cb-4a75-970b-04c1bbb6ffd6)

**Comment by OzGav** [comment]
Not yet possible to change. I have changed this to a feature request. See how many others vote.

**Comment by stouten** [comment]
Fully agree, would be great to have this as a feature in the future. I have been blasting loud music already many times because of this. 

-> would like to add to this feature request the possibility to see what the volume is when using the slider. As now it is always some sort of a gamble to see what the volume will be when the slider is released. Maybe something similar as in HA, see image below

![image](https://github.com/user-attachments/assets/deab22f0-18dd-48eb-8e30-7ae4a83662a5)

**Reply by askforblaine** [reply]
HA also does some weird stuff with sliders that doesn't translate well to mobile. (Tapping anywhere on the line causes a change)
Two apps offhand which handle this case well: Apple Music, Spotify

**Comment by askforblaine** [comment]
I also wrote about the problem here: https://github.com/music-assistant/support/issues/3861

**Comment by OzGav** [comment]
This will be satisifed by https://github.com/music-assistant/frontend/pull/1173
